cpu/esp32/periph/i2c_hw.c

```diff
@@ -147,7 +147,7 @@ void i2c_init(i2c_t dev)
     cfg.scl_io_num = i2c_config[dev].scl;
     cfg.sda_pullup_en = i2c_config[dev].sda_pullup;
     cfg.scl_pullup_en = i2c_config[dev].scl_pullup;
-#if SOC_I2C_SUPPORT_RTC
+#if defined(SOC_I2C_SUPPORT_RTC) && !defined(CPU_FAM_ESP32S3)
```

Comment:
```suggestion
#if defined(SOC_I2C_SUPPORT_RTC) && defined(I2C_SCLK_SRC_FLAG_LIGHT_SLEEP)
```

Would this work?

Reply:
No, `I2C_SCLK_SRC_FLAG_LIGHT_SLEEP` is just a definition of a flag which is defined always just by including the header file.